### ISSUE TITLE
"SCRIPT HOOK V CRITICAL ERROR"

### ISSUE THREAD

**Issue by Raging-Beard** [LukeRoss00/gta5-real-mod#315]
Hi, I am using a fresh install and getting this error. There doesn't seem to be a later version of Scripthook available, is something else at fault please? Thank you.

i9 10900k
3090
Reverb G2
Game 1.0.2215.0
Mod release 7
All steps follow, H selected, no GFX changes made or any other mods installed.

[Grand Theft Auto V.zip](https://github.com/LukeRoss00/gta5-real-mod/files/5958235/Grand.Theft.Auto.V.zip)

**Comment by Raging-Beard** [comment]
Sorry if this is just a case of waiting for a SH update (if there is one), I just wanted to check if there was something else at fault. Thanks.

**Comment by LukeRoss00** [comment]
Hey Beard, don't worry, it's not on your side. It's simply another Wednesday, another GTA V update that adds absolutely nothing to single player, but never fails to break Script Hook :-(

You can either follow the instructions [here](https://www.youtube.com/watch?v=YjsuNdhKgKY) to downgrade or wait for a Script Hook V update to be released, which usually takes a day or two

**Comment by Lightfoot86** [comment]
Does GTA update literally every single week?

**Comment by supernelsonz** [comment]
So do you just wait for an update or will there be new files to install? I just want to say the mod is life changing, its literally a 'wow' every minute experience, bravo sir!!

**Comment by LukeRoss00** [comment]
@Lightfoot86 I was joking, but for a game this old there are certainly lots of updates ;-)

@supernelsonz Thank you! Once the updated `ScriptHookV.dll` comes out, you'll have to use it to overwrite the old one in the game folder that no longer works.

**Comment by supernelsonz** [comment]
> @Lightfoot86 I was joking, but for a game this old there are certainly lots of updates ;-) The previous one was just before Christmas IIRC.
> 
> @supernelsonz Thank you! Once the updated `ScriptHookV.dll` comes out, you'll have to use it to overwrite the old one in the game folder that no longer works. The page to monitor is http://www.dev-c.com/gtav/scripthookv/

Thank you very much for the info and keep up the good work, you are clearly a genius!

**Comment by Obramm** [comment]
For anyone still getting this error download the file from [http://www.dev-c.com/gtav/scripthookv/] and replace the scripthookv.dll as the one on this page hasnt been updated.

**Comment by supernelsonz** [comment]
> @Lightfoot86 I was joking, but for a game this old there are certainly lots of updates ;-) The previous one was just before Christmas IIRC.
> 
> @supernelsonz Thank you! Once the updated `ScriptHookV.dll` comes out, you'll have to use it to overwrite the old one in the game folder that no longer works. The page to monitor is http://www.dev-c.com/gtav/scripthookv/

Hey Luke, I updated the new scripthook form Feb and it is still showing error, is anyone else getting this or am i doing something wrong?

**Comment by LukeRoss00** [comment]
@supernelsonz It works for me and other people. Did you actually overwrite the old `ScriptHookV.dll` in the game folder? That's where the dll should be, not in some `bin` subfolder for instance

**Comment by justinriddiough** [comment]
I seem to be getting this error.  I do have natural vision mod installed and not sure how to uninstall all the mods.

**Comment by justinriddiough** [comment]
Hmm. I think it was just tricky to get the new file in place. I copied it into the directory a number of times, and some of the times it was saying I needed admin privileges, other times it seemed to copy okay. Finally I deleted the original version, and double checked the timestamp from the archive with the directory, and it worked.  

So, maybe some people may experience similar, where they think they have copied it and windows doesn't give any real indication that it wasn't updated.
